mlflow/pyfunc/backend.py

```diff
@@ -318,10 +327,16 @@ def serve_stdin(
         model_uri,
         stdout=None,
         stderr=None,
+        model_config=None,
     ):
         local_path = _download_artifact_from_uri(model_uri)
+
+        command_env = os.environ.copy()
+        _set_mlflow_config_env(command_env, model_config)
+
         return self.prepare_env(local_path).execute(
             command=f"python {_STDIN_SERVER_SCRIPT} --model-uri {local_path}",
+            command_env=command_env,
```

Comment:
Can we test this? I thought it wasn't captured in the unittests

Reply:
sure. :)